manual/pom.xml

```diff
@@ -243,6 +243,20 @@
                     </exclusion>
                 </exclusions>
             </dependency>
+            <dependency>
```

Comment:
This should not be needed here (it's only intended for actual plugin implementations)